incubator/hnc/docs/user-guide/how-to.md

```diff
@@ -44,8 +44,11 @@ To install the plugin (Linux-only), first switch to a directory on your `PATH`
 # shown below, as of August 2020.
 HNC_VERSION=v0.5.1
 
+# Seletct the OS type where you run the plugin. Currently, you can choose "linux" or "darwin" (Mac).
+HNC_OS=linux
```

Comment:
I'd prefer to include the architecture here, especially since "darwin" will very shortly have multiple variants (amd64 vs arm). Maybe `HNC_PLATFORM=linux_amd64`?

Also, we should specify that this is only applies to versions _after_ 0.5.1. For 0.5.1, there's only the single release.

Reply:
> Maybe HNC_PLATFORM=linux_amd64?

Ok. Let me change to this.

> Also, we should specify that this is only applies to versions after 0.5.1. For 0.5.1, there's only the single release.

Ok. So let me add the separated installation instruction for pre 0.5.1 and after 0.5.1